packages/component-library/components/OffCanvas/OffCanvas.tsx

```diff
@@ -128,4 +128,6 @@ const withTrigger = (Component: React.ComponentType<any>) =>
     }
   }
 
-export default withContextProvider(withTrigger(OffCanvas))
+export default withDeprecatedComponent(OffCanvas, {
```

Comment:
I believe you'll want to continue to wrap `OffCavas` with the `withTrigger` HOC as you're now changing the component that's exported. Unless there was a specific reason for doing it this way?

Reply:
Oops I think this was a mistake, nice catch!